src/pages/RFOX/components/AddressSelection.tsx

```diff
@@ -8,10 +8,14 @@ import {
   Select,
   Stack,
 } from '@chakra-ui/react'
-import { useCallback, useMemo, useState } from 'react'
+import { type FC, useCallback, useMemo, useState } from 'react'
 import { useTranslate } from 'react-polyglot'
 
-export const AddressSelection = () => {
+type AddressSelectionProps = {
+  isNewAddress?: boolean
+}
+
```

Comment:
@0xApotheosis FYI, will `checkout --ours` the version from #6925 when mergefixing https://github.com/shapeshift/web/pull/6925/files#diff-9348e60af9e808feb1a6a737d4f2de927b5e5d2c3fd5e484aa9e26fbb4094cb9R19-R31

Absolutely fine as a dumb placeholder logic for now, but we already have fully wired up logic for account/custom address handling (will only need to add an obvious filtering out of the *current* address in the account dropdown, and form-based validation of custom address not being the current one)

Reply:
Ah nice, I hadn't seen that in-flight work - that's awesome, use "ours" on merge for sure!